GVFS/GVFS.Platform.POSIX/POSIXPlatform.cs

```diff
@@ -186,7 +185,7 @@ public override bool TryKillProcessTree(int processId, out int exitCode, out str
         [DllImport("libc", EntryPoint = "getuid", SetLastError = true)]
         private static extern int Getuid();
 
-        public class POSIXPlatformConstants : GVFSPlatformConstants
+        public abstract class POSIXPlatformConstants : GVFSPlatformConstants

```

Comment:
There should be no need to instantiate `POSIXPlatformConstants`, only subclasses which specify total behaviour. Accordingly I've marked it abstract and removed the `NotImplementedException` implementations for those which are overrided.